test/integration/fixtures/markdown-out.html

```diff
@@ -7,8 +7,7 @@ <h1>This is a heading with <em>italic</em></h1>
 <!-- /wp:paragraph -->
 
 <!-- wp:paragraph -->
-<p>Preserve<br>
-line breaks please.</p>
+<p>Preserve<br>line breaks please.</p>
```

Comment:
Changes seem in contrast with what was expected by this test. My understanding of Markdown is that it shouldn't make any meaningful difference. Yet, the test did exist presumably for some reason?

Reply:
@aduth This is the output HTML, which is semantically and visually the same. There's no difference except for a line break in the HTML source.

Reply:
In other words, it still does what is expected.